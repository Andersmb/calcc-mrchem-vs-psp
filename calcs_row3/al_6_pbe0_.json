{
  "input": {
    "molecule": {
      "cavity_coords": [
        {
          "center": [
            0.0,
            0.0,
            -1.2917784560432217
          ],
          "radius": 1.82
        },
        {
          "center": [
            0.0,
            0.0,
            1.8345842239567784
          ],
          "radius": 0.57
        }
      ],
      "cavity_width": 0.2,
      "charge": 0,
      "coords": [
        {
          "atom": "al",
          "xyz": [
            0.0,
            0.0,
            -1.2917784560432217
          ]
        },
        {
          "atom": "f",
          "xyz": [
            0.0,
            0.0,
            1.8345842239567784
          ]
        }
      ],
      "multiplicity": 1
    },
    "mpi": {
      "bank_size": 1,
      "numerically_exact": false,
      "shared_memory_size": 10000
    },
    "mra": {
      "basis_order": 9,
      "basis_type": "interpolating",
      "boxes": [
        2,
        2,
        2
      ],
      "corner": [
        -1,
        -1,
        -1
      ],
      "max_scale": 20,
      "min_scale": -5
    },
    "printer": {
      "file_name": "al_6_pbe0_.inp",
      "print_level": 0,
      "print_mpi": false,
      "print_prec": 6,
      "print_width": 75
    },
    "rsp_calculations": {},
    "scf_calculation": {
      "fock_operator": {
        "coulomb_operator": {
          "poisson_prec": 1e-06,
          "shared_memory": true
        },
        "exchange_operator": {
          "exchange_prec": -1.0,
          "poisson_prec": 1e-06
        },
        "kinetic_operator": {
          "derivative": "abgv_55"
        },
        "nuclear_operator": {
          "proj_prec": 1e-06,
          "shared_memory": true,
          "smooth_prec": 1e-06
        },
        "xc_operator": {
          "shared_memory": false,
          "xc_functional": {
            "cutoff": 0.0,
            "functionals": [
              {
                "coef": 1.0,
                "name": "pbe0"
              }
            ],
            "spin": true
          }
        }
      },
      "initial_guess": {
        "file_CUBE_a": "cube_vectors/CUBE_a_vector.json",
        "file_CUBE_b": "cube_vectors/CUBE_b_vector.json",
        "file_CUBE_p": "cube_vectors/CUBE_p_vector.json",
        "file_basis": "initial_guess/mrchem.bas",
        "file_chk": "checkpoint/phi_scf",
        "file_gto_a": "initial_guess/mrchem.moa",
        "file_gto_b": "initial_guess/mrchem.mob",
        "file_gto_p": "initial_guess/mrchem.mop",
        "file_phi_a": "initial_guess/phi_a_scf",
        "file_phi_b": "initial_guess/phi_b_scf",
        "file_phi_p": "initial_guess/phi_p_scf",
        "localize": true,
        "method": "DFT (PBE0)",
        "prec": 0.0001,
        "relativity": "None",
        "restricted": false,
        "screen": 12.0,
        "type": "sad_gto",
        "zeta": 0
      },
      "properties": {
        "dipole_moment": {
          "dip-1": {
            "operator": "h_e_dip",
            "precision": 1e-06,
            "r_O": [
              0.0,
              0.0,
              0.0
            ]
          }
        }
      },
      "scf_solver": {
        "checkpoint": false,
        "derivative": "abgv_55",
        "energy_thrs": 1e-06,
        "file_chk": "checkpoint/phi_scf",
        "final_prec": 1e-06,
        "helmholtz_prec": -1.0,
        "kain": 10,
        "light_speed": -1.0,
        "localize": true,
        "max_iter": 200,
        "method": "DFT (PBE0)",
        "orbital_thrs": 4.9999999999999996e-05,
        "proj_prec": 1e-06,
        "relativity": "None",
        "rotation": 0,
        "shared_memory": true,
        "smooth_prec": 1e-06,
        "start_prec": 1e-06
      },
      "write_orbitals": {
        "file_phi_a": "orbitals/phi_a_scf",
        "file_phi_b": "orbitals/phi_b_scf",
        "file_phi_p": "orbitals/phi_p_scf"
      }
    },
    "schema_name": "mrchem_input",
    "schema_version": 1
  },
  "output": {
    "properties": {
      "center_of_mass": [
        0.0,
        0.0,
        -8.339167003235483e-17
      ],
      "charge": 0,
      "dipole_moment": {
        "dip-1": {
          "magnitude": 0.548974905602018,
          "r_O": [
            0.0,
            0.0,
            0.0
          ],
          "vector": [
            6.1102331009965535e-06,
            -8.817006115947629e-05,
            -0.5489748984875811
          ],
          "vector_el": [
            6.110110514957988e-06,
            -8.81701837492055e-05,
            -0.26711298554463747
          ],
          "vector_nuc": [
            1.225860385650182e-10,
            1.2258972920227216e-10,
            -0.2818619129429436
          ]
        }
      },
      "geometry": [
        {
          "symbol": "Al",
          "xyz": [
            0.0,
            0.0,
            -1.2917784560432217
          ]
        },
        {
          "symbol": "F",
          "xyz": [
            0.0,
            0.0,
            1.8345842239567784
          ]
        }
      ],
      "multiplicity": 1,
      "orbital_energies": {
        "energy": [
          -56.169228365753504,
          -24.726467436187814,
          -4.1952910743093605,
          -2.7254707133519083,
          -2.725470708807389,
          -2.7212527542307545,
          -1.171546960201398,
          -0.47460547257658514,
          -0.4223725419780613,
          -0.42237252292382804,
          -0.2630005787521066,
          -56.16922836580672,
          -24.726467435975213,
          -4.195291074342361,
          -2.7254707133384253,
          -2.725470708495205,
          -2.72125275667894,
          -1.17154695939669,
          -0.474605469040483,
          -0.42237254291266807,
          -0.42237252463756203,
          -0.26300058142251115
        ],
        "occupation": [
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0
        ],
        "spin": [
          "a",
          "a",
          "a",
          "a",
          "a",
          "a",
          "a",
          "a",
          "a",
          "a",
          "a",
          "b",
          "b",
          "b",
          "b",
          "b",
          "b",
          "b",
          "b",
          "b",
          "b",
          "b"
        ],
        "sum_occupied": -192.0341582611195
      },
      "scf_energy": {
        "E_ee": 200.96610136726238,
        "E_eext": 0.0,
        "E_el": -379.61259269392997,
        "E_en": -893.189235649754,
        "E_kin": 341.52563950991674,
        "E_next": 0.0,
        "E_nn": 37.4236811194279,
        "E_nuc": 37.4236811194279,
        "E_tot": -342.1889115745021,
        "E_x": -7.050248962951922,
        "E_xc": -21.86484895840321,
        "Er_el": 0.0,
        "Er_nuc": 0.0,
        "Er_tot": 0.0
      }
    },
    "provenance": {
      "creator": "MRChem",
      "mpi_processes": 30,
      "nthreads": 6,
      "routine": "mrchem.x",
      "total_cores": 180,
      "version": "1.1.0-alpha"
    },
    "rsp_calculations": null,
    "scf_calculation": {
      "initial_energy": {
        "E_ee": 201.9909532773683,
        "E_eext": 0.0,
        "E_el": -377.6230207292719,
        "E_en": -890.2440278673024,
        "E_kin": 339.661677212481,
        "E_next": 0.0,
        "E_nn": 37.4236811194279,
        "E_nuc": 37.4236811194279,
        "E_tot": -340.199339609844,
        "E_x": -7.088138891390829,
        "E_xc": -21.943484460427964,
        "Er_el": 0.0,
        "Er_nuc": 0.0,
        "Er_tot": 0.0
      },
      "scf_solver": {
        "converged": true,
        "cycles": [
          {
            "energy_terms": {
              "E_ee": 199.38082585707494,
              "E_eext": 0.0,
              "E_el": -379.5192693572842,
              "E_en": -887.1203109393377,
              "E_kin": 336.91219338009273,
              "E_next": 0.0,
              "E_nn": 37.4236811194279,
              "E_nuc": 37.4236811194279,
              "E_tot": -342.0955882378563,
              "E_x": -6.9974444904596815,
              "E_xc": -21.694533164654466,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -342.0955882378563,
            "energy_update": 1.8962486280123017,
            "mo_residual": 0.3146026177049199,
            "wall_time": 92.794052407
          },
          {
            "energy_terms": {
              "E_ee": 201.14857084280757,
              "E_eext": 0.0,
              "E_el": -379.5935511289125,
              "E_en": -891.9867640916489,
              "E_kin": 340.1682843272637,
              "E_next": 0.0,
              "E_nn": 37.4236811194279,
              "E_nuc": 37.4236811194279,
              "E_tot": -342.1698700094846,
              "E_x": -7.054638777089173,
              "E_xc": -21.869003430245684,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -342.1698700094846,
            "energy_update": 0.07428177162830707,
            "mo_residual": 0.1508500187158699,
            "wall_time": 66.974878182
          },
          {
            "energy_terms": {
              "E_ee": 200.48799539107236,
              "E_eext": 0.0,
              "E_el": -379.60535639989496,
              "E_en": -891.7708614561527,
              "E_kin": 340.5285328978083,
              "E_next": 0.0,
              "E_nn": 37.4236811194279,
              "E_nuc": 37.4236811194279,
              "E_tot": -342.18167528046706,
              "E_x": -7.034520423683906,
              "E_xc": -21.816502808939006,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -342.18167528046706,
            "energy_update": 0.011805270982449656,
            "mo_residual": 0.09826658872121537,
            "wall_time": 59.596242131
          },
          {
            "energy_terms": {
              "E_ee": 200.87107127836538,
              "E_eext": 0.0,
              "E_el": -379.61078211135725,
              "E_en": -892.6216731938553,
              "E_kin": 341.0394734108957,
              "E_next": 0.0,
              "E_nn": 37.4236811194279,
              "E_nuc": 37.4236811194279,
              "E_tot": -342.18710099192936,
              "E_x": -7.046841242603255,
              "E_xc": -21.852812364159792,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -342.18710099192936,
            "energy_update": 0.005425711462294203,
            "mo_residual": 0.04725806098841661,
            "wall_time": 57.992794774
          },
          {
            "energy_terms": {
              "E_ee": 200.99623719517632,
              "E_eext": 0.0,
              "E_el": -379.6123274320215,
              "E_en": -893.2660781927858,
              "E_kin": 341.57608084750746,
              "E_next": 0.0,
              "E_nn": 37.4236811194279,
              "E_nuc": 37.4236811194279,
              "E_tot": -342.1886463125936,
              "E_x": -7.051005321812269,
              "E_xc": -21.867561960107203,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -342.1886463125936,
            "energy_update": 0.0015453206642632722,
            "mo_residual": 0.027049320828650105,
            "wall_time": 57.748625849
          },
          {
            "energy_terms": {
              "E_ee": 200.9832543520811,
              "E_eext": 0.0,
              "E_el": -379.61252218257073,
              "E_en": -893.3336042353415,
              "E_kin": 341.6555771992551,
              "E_next": 0.0,
              "E_nn": 37.4236811194279,
              "E_nuc": 37.4236811194279,
              "E_tot": -342.18884106314283,
              "E_x": -7.050658740127738,
              "E_xc": -21.867090758437676,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -342.18884106314283,
            "energy_update": 0.0001947505492125856,
            "mo_residual": 0.012415525141995724,
            "wall_time": 58.7644248
          },
          {
            "energy_terms": {
              "E_ee": 200.96967546293303,
              "E_eext": 0.0,
              "E_el": -379.61257301565786,
              "E_en": -893.2394409683934,
              "E_kin": 341.5726633544558,
              "E_next": 0.0,
              "E_nn": 37.4236811194279,
              "E_nuc": 37.4236811194279,
              "E_tot": -342.18889189622996,
              "E_x": -7.050206830220008,
              "E_xc": -21.865264034433263,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -342.18889189622996,
            "energy_update": 5.0833087129831256e-05,
            "mo_residual": 0.004908608978444269,
            "wall_time": 58.369789568
          },
          {
            "energy_terms": {
              "E_ee": 200.96821852545233,
              "E_eext": 0.0,
              "E_el": -379.612589166251,
              "E_en": -893.1893766244273,
              "E_kin": 341.5235503647591,
              "E_next": 0.0,
              "E_nn": 37.4236811194279,
              "E_nuc": 37.4236811194279,
              "E_tot": -342.1889080468231,
              "E_x": -7.050180866988684,
              "E_xc": -21.864800565046497,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -342.1889080468231,
            "energy_update": 1.6150593125985324e-05,
            "mo_residual": 0.0031170224121793845,
            "wall_time": 59.708918281
          },
          {
            "energy_terms": {
              "E_ee": 200.96757950248934,
              "E_eext": 0.0,
              "E_el": -379.6125915899927,
              "E_en": -893.1835061075478,
              "E_kin": 341.5183625500732,
              "E_next": 0.0,
              "E_nn": 37.4236811194279,
              "E_nuc": 37.4236811194279,
              "E_tot": -342.1889104705648,
              "E_x": -7.050221851421563,
              "E_xc": -21.864805683585846,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -342.1889104705648,
            "energy_update": 2.4237417051153898e-06,
            "mo_residual": 0.0012766618690664765,
            "wall_time": 59.400900989
          },
          {
            "energy_terms": {
              "E_ee": 200.96578883371518,
              "E_eext": 0.0,
              "E_el": -379.61259246882184,
              "E_en": -893.1838283421042,
              "E_kin": 341.5204225387946,
              "E_next": 0.0,
              "E_nn": 37.4236811194279,
              "E_nuc": 37.4236811194279,
              "E_tot": -342.18891134939395,
              "E_x": -7.050220207632427,
              "E_xc": -21.86475529159499,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -342.18891134939395,
            "energy_update": 8.788291552264127e-07,
            "mo_residual": 0.0007363547452655054,
            "wall_time": 60.071360886
          },
          {
            "energy_terms": {
              "E_ee": 200.96533929292886,
              "E_eext": 0.0,
              "E_el": -379.6125926210786,
              "E_en": -893.185470425835,
              "E_kin": 341.5225180253182,
              "E_next": 0.0,
              "E_nn": 37.4236811194279,
              "E_nuc": 37.4236811194279,
              "E_tot": -342.1889115016507,
              "E_x": -7.050222571482172,
              "E_xc": -21.86475694200852,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -342.1889115016507,
            "energy_update": 1.5225674587782123e-07,
            "mo_residual": 0.0002961146533802552,
            "wall_time": 63.08923277
          },
          {
            "energy_terms": {
              "E_ee": 200.96571031786388,
              "E_eext": 0.0,
              "E_el": -379.6125926782157,
              "E_en": -893.1879720915915,
              "E_kin": 341.52472153276875,
              "E_next": 0.0,
              "E_nn": 37.4236811194279,
              "E_nuc": 37.4236811194279,
              "E_tot": -342.1889115587878,
              "E_x": -7.050239754406866,
              "E_xc": -21.86481268285006,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -342.1889115587878,
            "energy_update": 5.713712880606181e-08,
            "mo_residual": 0.00021079716497243633,
            "wall_time": 65.350144122
          },
          {
            "energy_terms": {
              "E_ee": 200.96610136726238,
              "E_eext": 0.0,
              "E_el": -379.61259269392997,
              "E_en": -893.189235649754,
              "E_kin": 341.52563950991674,
              "E_next": 0.0,
              "E_nn": 37.4236811194279,
              "E_nuc": 37.4236811194279,
              "E_tot": -342.1889115745021,
              "E_x": -7.050248962951922,
              "E_xc": -21.86484895840321,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -342.1889115745021,
            "energy_update": 1.5714249457232654e-08,
            "mo_residual": 8.825584652755502e-05,
            "wall_time": 75.959128608
          }
        ],
        "wall_time": 835.87257229
      },
      "success": true
    },
    "schema_name": "mrchem_output",
    "schema_version": 1,
    "success": true
  }
}
