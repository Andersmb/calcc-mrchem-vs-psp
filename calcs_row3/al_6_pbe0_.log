Starting job 5688105 on c1-[27-28,30,53],c5-2 at Thu May 19 19:38:26 CEST 2022

Copying al_6_pbe0_.out to /cluster/home/ambr/projects/psp/row3/calcs/singlets
Copying al_6_pbe0_.json to /cluster/home/ambr/projects/psp/row3/calcs/singlets

Task and CPU usage stats:
JobID           JobName  AllocCPUS   NTasks     MinCPU MinCPUTask     AveCPU    Elapsed ExitCode 
------------ ---------- ---------- -------- ---------- ---------- ---------- ---------- -------- 
5688105      al_6_pbe0_        180                                             00:15:34      0:0 
5688105.bat+      batch         36        1   00:00:02          0   00:00:02   00:15:34      0:0 
5688105.ext+     extern        180        5   00:00:00          4   00:00:00   00:15:34      0:0 
5688105.0      mrchem.x        180       30   00:15:11         29   00:40:13   00:15:14      0:0 

Memory usage stats:
JobID            MaxRSS MaxRSSTask     AveRSS MaxPages   MaxPagesTask   AvePages 
------------ ---------- ---------- ---------- -------- -------------- ---------- 
5688105                                                                          
5688105.bat+     19280K          0     19280K        0              0          0 
5688105.ext+          0          4          0        0              4          0 
5688105.0      7185452K         29 3291485525        0             23          0 

Disk usage stats:
JobID         MaxDiskRead MaxDiskReadTask    AveDiskRead MaxDiskWrite MaxDiskWriteTask   AveDiskWrite 
------------ ------------ --------------- -------------- ------------ ---------------- -------------- 
5688105                                                                                               
5688105.bat+       19.39M               0         19.39M        0.06M                0          0.06M 
5688105.ext+        0.00M               4          0.00M            0                4              0 
5688105.0           6.77M              14          6.74M       63.09M               14         42.32M 

Job 5688105 completed at Thu May 19 19:54:00 CEST 2022
