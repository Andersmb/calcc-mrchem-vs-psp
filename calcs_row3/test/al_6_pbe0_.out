                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                                                                     ***
***          __  __ ____   ____ _                                       ***
***         |  \/  |  _ \ / ___| |__   ___ _ __ ___                     ***
***         | |\/| | |_) | |   | '_ \ / _ \ '_ ` _ \                    ***
***         | |  | |  _ <| |___| | | |  __/ | | | | |                   ***
***         |_|  |_|_| \_\\____|_| |_|\___|_| |_| |_|                   ***
***                                                                     ***
***         VERSION            1.1.0-alpha                              ***
***                                                                     ***
***         Git branch         HEAD                                     ***
***         Git commit hash    e7d4b230bb6df8495b9a                     ***
***         Git commit author  Stig Rune Jensen                         ***
***         Git commit date    Sat Mar 19 14:14:51 2022 +0100           ***
***                                                                     ***
***         Contact: luca.frediani@uit.no                               ***
***                                                                     ***
***         Radovan Bast            Magnar Bjorgve                      ***
***         Roberto Di Remigio      Antoine Durdek                      ***
***         Luca Frediani           Gabriel Gerez                       ***
***         Stig Rune Jensen        Jonas Juselius                      ***
***         Rune Monstad            Peter Wind                          ***
***                                                                     ***
***************************************************************************

---------------------------------------------------------------------------

 MPI processes           :        (1 bank)                              30
 OpenMP threads          :                                               6
 Total cores             :                                             175
                                                                           
---------------------------------------------------------------------------

XCFun DFT library Copyright 2009-2020 Ulf Ekstrom and contributors.
See http://dftlibs.org/xcfun/ for more information.

This is free software; see the source code for copying conditions.
There is ABSOLUTELY NO WARRANTY; not even for MERCHANTABILITY or
FITNESS FOR A PARTICULAR PURPOSE. For details see the documentation.
Scientific users of this library should cite
U. Ekstrom, L. Visscher, R. Bast, A. J. Thorvaldsen and K. Ruud;
J.Chem.Theor.Comp. 2010, DOI: 10.1021/ct100117s

---------------------------------------------------------------------------

 MRCPP version         : 1.4.1
 Git branch            : HEAD
 Git commit hash       : 75d41879b1908a94a452
 Git commit author     : Stig Rune Jensen
 Git commit date       : Thu Jan 6 11:38:53 2022 +0100

 Linear algebra        : EIGEN v3.4.0
 Parallelization       : MPI/OpenMP (6 threads)

---------------------------------------------------------------------------


                                                                           
===========================================================================
                         MultiResolution Analysis
---------------------------------------------------------------------------
 polynomial order      : 9
 polynomial type       : Interpolating
---------------------------------------------------------------------------
 total boxes           : 8
 boxes                 : [          2           2           2 ]
 unit lengths          : [   32.00000    32.00000    32.00000 ]
 scaling factor        : [    1.00000     1.00000     1.00000 ]
 lower bounds          : [  -32.00000   -32.00000   -32.00000 ]
 upper bounds          : [   32.00000    32.00000    32.00000 ]
 total length          : [   64.00000    64.00000    64.00000 ]
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                        Initializing Molecule                        ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0      Al            :        0.000000        0.000000       -1.291778
    1       F            :        0.000000        0.000000        1.834584
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000       -0.000000
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                 Computing Ground State Wavefunction                 ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial orbitals
 Method                  : Diagonalize SAD Hamiltonian
 Precision               : 1.00000e-04
 Screening               : 1.20000e+01 StdDev
 Restricted              : False
 Functional              : LDA (SVWN5)
 AO basis                : 3-21G
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                            Molecular Orbitals
---------------------------------------------------------------------------
 Alpha electrons         :                                              11
 Beta electrons          :                                              11
 Total electrons         :                                              22
---------------------------------------------------------------------------
    n  Occ Spin          :                                            Norm
---------------------------------------------------------------------------
    0    1    a          :                              9.999999092522e-01
    1    1    a          :                              9.999999529198e-01
    2    1    a          :                              9.999999571625e-01
    3    1    a          :                              1.000000007974e+00
    4    1    a          :                              9.999999812284e-01
    5    1    a          :                              9.999999863152e-01
    6    1    a          :                              1.000000192279e+00
    7    1    a          :                              9.999997480617e-01
    8    1    a          :                              1.000000641163e+00
    9    1    a          :                              1.000000655280e+00
   10    1    a          :                              9.999936726818e-01
   11    1    b          :                              1.000000033682e+00
   12    1    b          :                              9.999999530013e-01
   13    1    b          :                              9.999999573951e-01
   14    1    b          :                              1.000000007994e+00
   15    1    b          :                              9.999999812284e-01
   16    1    b          :                              9.999999877410e-01
   17    1    b          :                              1.000000192388e+00
   18    1    b          :                              9.999997487611e-01
   19    1    b          :                              1.000000640307e+00
   20    1    b          :                              1.000000659378e+00
   21    1    b          :                              9.999997367916e-01
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial energy
 Method                  : DFT (PBE0)
 Relativity              : None
 Precision               : 1.00000e-04
 Localization            : On
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                        Molecular Energy (initial)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                339.661677212481
 E-N energy              :            (au)               -890.244027867302
 Coulomb energy          :            (au)                201.990953277368
 Exchange energy         :            (au)                 -7.088138891391
 X-C energy              :            (au)                -21.943484460428
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                 37.423681119428
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)               -377.623020729272
 Nuclear energy          :            (au)                 37.423681119428
---------------------------------------------------------------------------
 Total energy            :            (au)             -3.401993396098e+02
                         :      (kcal/mol)             -2.134783069527e+05
                         :        (kJ/mol)             -8.931932368699e+05
                         :            (eV)             -9.257295223879e+03
===========================================================================


===========================================================================
                        Orbital Energies (initial)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)                -55.805769410447
    1    1    a          :            (au)                -24.305827164025
    2    1    a          :            (au)                 -4.183317773545
    3    1    a          :            (au)                 -2.709661145729
    4    1    a          :            (au)                 -2.709619881897
    5    1    a          :            (au)                 -2.705249013118
    6    1    a          :            (au)                 -1.025366466241
    7    1    a          :            (au)                 -0.347555421656
    8    1    a          :            (au)                 -0.261190817821
    9    1    a          :            (au)                 -0.260876359898
   10    1    a          :            (au)                 -0.229042505591
   11    1    b          :            (au)                -55.805828586071
   12    1    b          :            (au)                -24.305827189544
   13    1    b          :            (au)                 -4.183318026558
   14    1    b          :            (au)                 -2.709661459481
   15    1    b          :            (au)                 -2.709620154481
   16    1    b          :            (au)                 -2.705249067944
   17    1    b          :            (au)                 -1.025366030536
   18    1    b          :            (au)                 -0.347559794519
   19    1    b          :            (au)                 -0.261190655473
   20    1    b          :            (au)                 -0.260876254047
   21    1    b          :            (au)                 -0.229048522613
---------------------------------------------------------------------------
 Sum occupied            :            (au)               -189.087021701235
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Optimize ground state orbitals
 Method                  : DFT (PBE0)
 Relativity              : None
 Checkpointing           : Off
 Max iterations          : 200
 KAIN solver             : 10
 Localization            : First two iterations
 Diagonalization         : Off
 Start precision         : 1.00000e-06
 Final precision         : 1.00000e-06
 Helmholtz precision     : Dynamic
 Energy threshold        : 1.00000e-06
 Orbital threshold       : 5.00000e-05
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
 Iter           MO residual             Total energy                Update
---------------------------------------------------------------------------
    0          4.690416e+00        -340.199339609844         -3.401993e+02
    1          3.146026e-01        -342.095588237856         -1.896249e+00
    2          1.508500e-01        -342.169870009485         -7.428177e-02
    3          9.826659e-02        -342.181675280467         -1.180527e-02
    4          4.725806e-02        -342.187100991929         -5.425711e-03
    5          2.704932e-02        -342.188646312594         -1.545321e-03
    6          1.241553e-02        -342.188841063143         -1.947505e-04
    7          4.908609e-03        -342.188891896230         -5.083309e-05
    8          3.117022e-03        -342.188908046823         -1.615059e-05
    9          1.276662e-03        -342.188910470565         -2.423742e-06
   10          7.363547e-04        -342.188911349394         -8.788292e-07
   11          2.961147e-04        -342.188911501651         -1.522567e-07
   12          2.107972e-04        -342.188911558788         -5.713713e-08
   13          8.825585e-05        -342.188911574502         -1.571425e-08
---------------------------------------------------------------------------
                      SCF converged in 13 iterations!
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                    Printing Molecular Properties                    ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0      Al            :        0.000000        0.000000       -1.291778
    1       F            :        0.000000        0.000000        1.834584
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000       -0.000000
===========================================================================


===========================================================================
                         Molecular Energy (final)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                341.525639509917
 E-N energy              :            (au)               -893.189235649754
 Coulomb energy          :            (au)                200.966101367262
 Exchange energy         :            (au)                 -7.050248962952
 X-C energy              :            (au)                -21.864848958403
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                 37.423681119428
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)               -379.612592693930
 Nuclear energy          :            (au)                 37.423681119428
---------------------------------------------------------------------------
 Total energy            :            (au)             -3.421889115745e+02
                         :      (kcal/mol)             -2.147267821998e+05
                         :        (kJ/mol)             -8.984168573071e+05
                         :            (eV)             -9.311434232694e+03
===========================================================================


===========================================================================
                         Orbital Energies (final)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)                -56.169228365754
    1    1    a          :            (au)                -24.726467436188
    2    1    a          :            (au)                 -4.195291074309
    3    1    a          :            (au)                 -2.725470713352
    4    1    a          :            (au)                 -2.725470708807
    5    1    a          :            (au)                 -2.721252754231
    6    1    a          :            (au)                 -1.171546960201
    7    1    a          :            (au)                 -0.474605472577
    8    1    a          :            (au)                 -0.422372541978
    9    1    a          :            (au)                 -0.422372522924
   10    1    a          :            (au)                 -0.263000578752
   11    1    b          :            (au)                -56.169228365807
   12    1    b          :            (au)                -24.726467435975
   13    1    b          :            (au)                 -4.195291074342
   14    1    b          :            (au)                 -2.725470713338
   15    1    b          :            (au)                 -2.725470708495
   16    1    b          :            (au)                 -2.721252756679
   17    1    b          :            (au)                 -1.171546959397
   18    1    b          :            (au)                 -0.474605469040
   19    1    b          :            (au)                 -0.422372542913
   20    1    b          :            (au)                 -0.422372524638
   21    1    b          :            (au)                 -0.263000581423
---------------------------------------------------------------------------
 Sum occupied            :            (au)               -192.034158261120
===========================================================================


===========================================================================
                           Dipole Moment (dip-1)
---------------------------------------------------------------------------
 r_O                     :        0.000000        0.000000        0.000000
---------------------------------------------------------------------------
 Electronic vector       :        0.000006       -0.000088       -0.267113
 Magnitude               :            (au)                        0.267113
                         :         (Debye)                        0.678933
---------------------------------------------------------------------------
 Nuclear vector          :        0.000000        0.000000       -0.281862
 Magnitude               :            (au)                        0.281862
                         :         (Debye)                        0.716421
---------------------------------------------------------------------------
 Total vector            :        0.000006       -0.000088       -0.548975
 Magnitude               :            (au)                        0.548975
                         :         (Debye)                        1.395355
===========================================================================


                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                            Exiting MRChem                           ***
***                                                                     ***
***                       Wall time :  0h 15m 12s                       ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
